### PR TITLE
Followup edits to cidr range pr 8955

### PR DESCRIPTION
**Pull request by bfallonf**
For: #8955 

Edits as suggested, with creating a module for "evacuating a node" and including it here.

cc @knobunc

### CODE REVIEW

**Review by knobunc — APPROVED** [review]
LGTM.  Thanks Brice

**Comment by bfallonf** [comment]
Thanks @knobunc . QA'd in original PR.

@openshift/team-documentation PTAL

File: install_config/configuring_sdn.adoc [hunk 1]
```diff
@@ -163,12 +163,11 @@ default value of *10.128.0.0/14*, you could change
 of net 10) but not to *10.64.0.0/16*, because that does not overlap
 the original value.
 
-You could change `*serviceNetworkCIDR*` from *172.30.0.0/16* to
-*172.30.0.0/15*, but not to *172.28.0.0/14* because even though the
-original range is entirely inside the new range, the original range
-must be at the start of the CIDR.  See
-[growing-the-service-network](growing the service network) for
-details.
+You could change `*serviceNetworkCIDR*` from *172.30.0.0/16* to *172.30.0.0/15*,
```
Comment (kalexand-rh) [inline install_config/configuring_sdn.adoc]: s/could/can

File: install_config/configuring_sdn.adoc [hunk 2]
```diff
@@ -192,26 +191,41 @@ networkConfig:
...
 ====
 
-[[growing-the-service-network]]
-== Growing the Service Network
+[[expanding-the-service-network]]
+== Expanding the service network
 
-If you are running low on addresses in your Service network and can expand the range so that it includes the current range at the beginning of the new range, then you can:
+If you are running low on addresses in your service network, you can expand the
+range, ensuring that the current range at the beginning of the new range:
```
Comment (kalexand-rh) [inline install_config/configuring_sdn.adoc]: s/are running/run

s/range, ensuring/range as long as you ensure

File: install_config/configuring_sdn.adoc [hunk 3]
```diff
@@ -192,26 +191,41 @@ networkConfig:
...
-. Restart the *atomic-openshift-master-api* and *atomic-openshift-master-controllers*
-on masters.
+----
+# systemctl restart atomic-openshift-master-controllers
+----
 endif::[]
-. Update the inventory file to change the openshift_portal_net to the new CIDR and run the playbook `*config.yml*` to redeploy the cluster
-. Evacuate each node, one by one, restart it, and then re-enable it
+
+. Update the value of the `openshift_portal_net` variable in the Ansible
```
Comment (kalexand-rh) [inline install_config/configuring_sdn.adoc]: I'd make this two steps and include the command to run the Ansible playbook.

**Review by kalexand-rh — COMMENTED** [review]
@bfallonf, I have a couple of comments, but this looks good.

File: install_config/configuring_sdn.adoc (hunk 3 above)
Comment (bfallonf) [reply install_config/configuring_sdn.adoc]: Hmm @knobunc which playbook are we referring to? The master one? So:

----
# ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-master/config.yml
----

**Comment by bfallonf** [comment]
Thanks @kalexand-rh . I've updated to suggestions except for one thing I have a question about. I'll get the info and update the rest.

File: install_config/configuring_sdn.adoc (hunk 3 above)
Comment (knobunc) [reply install_config/configuring_sdn.adoc]: Yes, I think that's the right one.  @sdodson is that correct?

Comment (sdodson) [reply install_config/configuring_sdn.adoc]: Yes that's correct.

Comment (bfallonf) [reply install_config/configuring_sdn.adoc]: Thanks @sdodson @knobunc ! I'll update and merge.

**Comment by bfallonf** [comment]
Updated. This has been through review, so I'll merge.

**Pull request merged**